### DISCUSSION TITLE
How do I go back if I made a wrong choice?

### DISCUSSION THREAD

**Discussion by thytrherfvsefasfd443sf** [archlinux/archinstall#1098] [Q&A] [answered]
What is the key to go back to the previous step? I could not find any, so I just pressed Ctrl+C and started again back from the beginning. There is no way? If so, can't it be easily implemented with some sort of go-to statement?

**Comment by Torxed** [comment, marked as answer]
In v2.4.0+ there's a new menu system that allows you to go back and change items :)
There's not really any go-to statements in Python, and the old logic didn't store states so it would have been hard :)

![image](https://user-images.githubusercontent.com/861439/160357312-5bc564da-0a69-4e5a-9a55-f81dc02bc023.png)
